modules/passkey/docker-compose.yaml

```diff
@@ -0,0 +1,37 @@
+version: '3.8'
+
+services:
+  geth:
+    image: docker.io/ethereum/client-go:stable
+    restart: always
+    environment:
+      GETH_DEV: 'true'
+      GETH_HTTP: 'true'
+      GETH_HTTP_ADDR: '0.0.0.0'
+      GETH_HTTP_API: 'personal,eth,net,web3,debug'
+      GETH_HTTP_VHOSTS: '*'
+      GETH_RPC_ALLOW_UNPROTECTED_TXS: 'true'
+    ports:
+      - 8545:8545
+
+  bundler:
+    build:
+      context: .
+      dockerfile: docker/bundler/Dockerfile
+      args:
+        TAG: 26e4f4c433916a6a2ea8ecc91ba8a56cd904f27f
+    restart: always
+    command: ['--auto', '--network=http://geth:8545']
+    ports:
+      - 3000:3000
+
+  bundler-upstream:
```

Comment:
nit: not needed for the passkey project IMO.

Reply:
I don't think it for any contract intended to work with 4337 it hurts to test the compatibility with the upstream